src/svg/path/path.py

```diff
@@ -49,6 +49,31 @@ def __ne__(self, other):
             return NotImplemented
         return not self == other
 
+    def __add__(self, other):
+        self.start += other
+        self.end += other
```

Comment:
This pretty much is the same as SVG translate, I don't mind having it as __add__ as well, but there should also be a translate() function. Perhaps translate() could take X, Y instead of a complex number?

Reply:
This is the same, though as a dunder method you get to do line + (2 + 4j) or whatever and have that merely work.